script/build.sh

```diff
@@ -55,10 +63,28 @@ cp -r $DIR/docs ${SRC_DIR}/pfe/portal/
 
 # Copy the appsody extension into portal. The zip file must have a version number e.g. codewind-appsody-extension-0.3.0.zip
 # in order for it to be accepted as a valid extension
-echo -e "\n+++   DOWNLOADING EXTENSIONS   +++\n";
+
 mkdir -p ${SRC_DIR}/pfe/extensions
 rm -f ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-*.zip
-curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-0.5.0.zip https://github.com/eclipse/codewind-appsody-extension/archive/0.5.0.zip
+
+DEFAULT_APPSODY_EXT_VERSION="0.5.0"
+
+if [ "$DEVMODE" == "true" ]; then
+  if [ "$APPSODY_EXT_VERSION" == "none" ]; then 
+    echo -e "\n+++   SKIPPING APPSODY EXTENSION DOWNLOAD DUE TO ENV VARIABLE APPSODY_EXT_VERSION=none +++\n";
+  else
+    if [ -z "$APPSODY_EXT_VERSION" ]; then 
+      APPSODY_EXT_VERSION=$DEFAULT_APPSODY_EXT_VERSION
+    fi
+
+    echo -e "\n+++   DOWNLOADING APPSODY EXTENSION: ${APPSODY_EXT_VERSION}  +++\n";
+    curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-${APPSODY_EXT_VERSION}.zip https://github.com/eclipse/codewind-appsody-extension/archive/${APPSODY_EXT_VERSION}.zip
```

Comment:
Could we avoid duplicating these lines by checking if appsody version is set?  (then `--dev APPSODY_EXT_VERSION=none` can unset the version to skip the lines)